src/interface.h

```diff
@@ -135,4 +135,58 @@ CK_RV side_channel_free_Decrypt(P11PROV_CTX *ctx, CK_SESSION_HANDLE hSession,
 
 CK_INFO p11prov_module_ck_info(P11PROV_MODULE *mctx);
 
+/* never blocked calls are masked to 0 by default */
+/* 2.40 */
```

Comment:
I think the values here deserve some more explanation:
 * whats the point adding the check for the calls that can never be blocked? Do we assume compiler will compile the check out?
 * Why the only value (other than 0) so far is `0b1000` (8) and not `1` to start with?

Reply:
The pre-generated code from interface.pre needs all the definitions, but we need to  assign an actual value only once we provide a knob to turn that value on/off

I thought about pre-assigning values too, but not sure that is useful.